src/app/components/LiveLabel/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface LiveLabelProps extends TextProps {
+  className?: string;
+}
+
+export interface TextProps {
+  offScreenText?: string;
+  lang?: string;
+  id?: string;
+}
```

Comment:
**nitpick** I assume the order of this doesn't matter i.e. TextProps don't need to be declared higher in the file before being used?

Reply:
Hmm yeah I'm not sure, I'll put it after just in case. I'm not getting any vscode compiler error hints saying anything is wrong

Reply:
Yeah I think we leave it, looks like it's able to cater for it regardless of its position in the file